app/src/main/java/com/oracle/javafx/scenebuilder/app/about/AboutWindowController.java

```diff
@@ -85,6 +88,7 @@ public AboutWindowController() {
                 sbBuildInfo = sbProps.getProperty("build.info", "UNSET"); //NOI18N
                 sbBuildVersion = sbProps.getProperty("build.version", "UNSET"); //NOI18N
                 sbBuildDate = sbProps.getProperty("build.date", "UNSET"); //NOI18N
+                sbBuildDateFormat = sbProps.getProperty("build.date.format", "UNSET"); //NOI18N
```

Comment:
why don't you use directly `maven.build.timestamp.format`?

Reply:
trying to keep things as they have been. `maven.build.timestamp` is not used directly and passed via `build.date` instead.